homeassistant/components/lightwave/__init__.py

```diff
@@ -50,4 +70,19 @@
             async_load_platform(hass, "switch", DOMAIN, switches, config)
         )
 
+    trv = config[DOMAIN][CONF_TRV]
+    trvs = trv[CONF_TRVS]
+    if trv:
+        proxy_ip = trv[PROXY_IP]
+        proxy_port = trv[PROXY_PORT]
+        hass.data[LIGHTWAVE_TRV_PROXY] = proxy_ip
+        hass.data[LIGHTWAVE_TRV_PROXY_PORT] = proxy_port
+        lwlink.set_trv_proxy(proxy_ip, proxy_port)
+        hass.async_create_task(
+            async_load_platform(hass, "climate", DOMAIN, trvs, config)
```

Comment:
Import the `DOMAIN` constant from climate/init.py rename it and use it here.

Reply:
Done